postgres/portal_app.go

```diff
@@ -0,0 +1,83 @@
+package postgres
```

Comment:
Resolving this with the understanding that references to `portal`, etc. are because that's what it's called in the Grove Portal and it's not the responsibility of this repo to make any changes in naming of structure to the existing Portal backend repos.